index.md

```diff
@@ -1,131 +1,198 @@
+countries met the challenge well and copy their systems and have our nations be similarly prepared.
 
-Having sufficient shelter and food for every family is also a human right. Too many governments spend money on defense and lavish projects while families go hungry and without shelter. We must hold governments accountable to provide basic human services for all of their citizens. 
+A baseline education must also be free for everyone. We should teach everyone the importance of science and math,
+but we must not forget teaching communication, languages and art. Basic education as well as university education
+must be free or easily affordable for everyone and focus on goals beyond turning people into workers. 
 
-We should spend more money on human health than we spend on defense and war and we should never let health care be for-profit.
+Having sufficient shelter and food for every family is also a human right. Too many governments spend money on 
+defense and lavish projects while families go hungry and without shelter. We must hold governments accountable 
+to provide basic human services for all of their citizens. 
 
+Health, education and shelter should never be for-profit services and we should spend more on these services
+than we spend on defense and war.
 
-Cooperation > Competition
+
+Cooperation > competition
 -------------------------
 
-We have enough resources in this world to cover everyone's basic needs. But drastic inequality prevents this from happening as some people hoard resources and money. We need to reduce the inequality and free up resources to cover everyone’s basic needs. Especially now during a pandemic, we need to freely share medical information and cooperate on treatments, vaccines and medical devices in order to save lives.
+We have enough resources in this world to cover everyone's basic needs. But drastic inequality prevents this 
+from happening as some people hoard resources and money. We need to reduce the inequality and free up resources 
+to cover everyone’s basic needs. Especially now during a pandemic, we need to freely share medical information and 
+cooperate on treatments, vaccines and medical devices in order to save lives.
 
-Competing for resources can often be wasteful. How can we focus on cooperating in order to cover the basic needs of all people? The open source movement that produces much of the software that makes the Internet possible is created via global cooperation. This cooperation consumes far fewer resources than a competitive model would consume. How can we cooperate more and compete less?
+Competing for resources can often be wasteful. How can we focus on cooperating in order to cover the basic needs 
+of all people? The open source movement that produces much of the software that makes the Internet possible is created 
+via global cooperation. This cooperation consumes far fewer resources than a competitive model would consume. How 
+can we cooperate more and compete less?
```

Comment:
I'd drop this question. If people accept "Cooperation > competition" as a basic principle, they'll find ways to do it.
Not that the question has no sense, it has, but just doesn't belong here imho. Also other paragraphs don't end with a question.